### DISCUSSION TITLE
build armeabi-v7a version apk for android

### DISCUSSION THREAD

**Discussion by Jzhenli** [beeware/briefcase#1946] [Q&A] [answered]
Hi, when build android apk, default version is arm64-v8a,  how to build armeabi-v7a version? Thanks

**Comment by mhsmith** [comment, marked as answer]
As shown in [the template](https://github.com/beeware/briefcase-android-gradle-template/blob/v0.3.19/%7B%7B%20cookiecutter.format%20%7D%7D/app/build.gradle#L33), armeabi-v7a is no longer supported with Python 3.12 and newer. Create a new virtual environment with Python 3.11 or older, and then rerun `briefcase create android`.
